rules.md

```diff
@@ -0,0 +1,18 @@
+# Substrate
+
+- 🔒️ed lines are reviewed by 2 team leads, one of them must be in @paritytech/locks-review and the other in @paritytech/polkadot-review.
+- Normal PRs are reviewed by 2 of @paritytech/core-devs. This team's reviews should not be requested, but the check should fail explaining this rule.
```

Comment:
```suggestion
- PRs that do not touch locked lines require 2 positive reviews from @paritytech/core-devs.
```

I think specifying the notification strategy is an implementation detail and not part of the rules. Same for the messaging on failure: a requirement for the implementation but not really part of the rule. 

Any failure should explain itself and with the current wording it almost sounds like the locked lines rule does not need to explain anything, while the normal-PR-rule does require it.